deployer/README.md

```diff
@@ -1175,3 +1175,249 @@ the first created. You can check if the route in question is defined in multiple
+        component: "fluentd"
+    spec:
+      replicas: 0
+      selector:
+        provider: openshift
+        component: "fluentd"
+      triggers: {}
+      strategy:
+        resources: {}
+        rollingParams:
+          intervalSeconds: 1
+          timeoutSeconds: 600
+          updatePeriodSeconds: 1
+        type: Recreate
+      template:
+        metadata:
+          name: enrichment-fluentd-elasticsearch
+          labels:
+            provider: openshift
+            component: "fluentd"
+        spec:
+          terminationGracePeriodSeconds: 300
+          serviceAccountName: aggregated-logging-fluentd
+          containers:
+          - name: enrichment-fluentd-elasticsearch
+            image: registry.access.redhat.com/openshift3/ose-logging-fluentd:latest
+            imagePullPolicy: Always
+            securityContext:
+              privileged: true
+            resources:
+              limits:
+                cpu: 100m
+            ports:
+              - containerPort: 1095
+                hostPort: 1095
+                protocol: TCP
+                purpose: "to prevent more than one per node"
+            volumeMounts:
+            - name: config
+              mountPath: /etc/fluent/configs.d/user
+              readOnly: true
+            - name: certs
+              mountPath: /etc/fluent/keys
+              readOnly: true
+            - name: dockerhostname
+              mountPath: /etc/docker-hostname
+              readOnly: true
+            - name: localtime
+              mountPath: /etc/localtime
+              readOnly: true
+            - name: dockercfg
+              mountPath: /etc/sysconfig/docker
+              readOnly: true
+            env:
+            - name: "K8S_HOST_URL"
+              value: https://kubernetes.default.svc.cluster.local
+            - name: "ES_HOST"
+              value: logging-es
+            - name: "ES_PORT"
+              value: "9200"
+            - name: "ES_CLIENT_CERT"
+              value: /etc/fluent/keys/cert
+            - name: "ES_CLIENT_KEY"
+              value: /etc/fluent/keys/key
+            - name: "ES_CA"
+              value: /etc/fluent/keys/ca
+            - name: "OPS_HOST"
+              value: <logging-es | logging-es-ops>
+            - name: "OPS_PORT"
+              value: "9200"
+            - name: "OPS_CLIENT_CERT"
+              value: /etc/fluent/keys/cert
+            - name: "OPS_CLIENT_KEY"
+              value: /etc/fluent/keys/key
+            - name: "OPS_CA"
+              value: /etc/fluent/keys/ca
+          volumes:
+          - name: config
+            configMap:
+              name: logging-fluentd-enrichment
+          - name: certs
+            secret:
+              secretName: logging-fluentd
+          - name: dockerhostname
+            hostPath:
+              path: /etc/hostname
+          - name: localtime
+            hostPath:
+              path: /etc/localtime
+          - name: dockercfg
```

Comment:
I think this mountpoint is used to understand whether journal log driver is used. It shouldn't be needed here imho.